acceptance/bundle/python/unicode-support/databricks.yml

```diff
@@ -0,0 +1,19 @@
+bundle:
+  name: my_project
+
+sync: { paths: [] } # don't need to copy files
+
+experimental:
+  python:
+    resources:
+      - "resources:load_resources"
+
+resources:
+  jobs:
+    job_1:
+      name: "🔥🔥🔥"
```

Comment:
nit: this tests unicode support in general, right? good to have, not specific to python, so I'd split this test.

Reply:
I think it isn't a problem for Golang code, because it's already using Unicode everywhere. It was just Python having a weird behavior